apps/costcontrol/style/views/datausage.css

```diff
@@ -18,35 +18,46 @@
 }
 
 #datausage-tab .toggle-data li {
-  padding-left: 3rem;
   height: 3.8rem;
+
+  -moz-padding-start: 3rem;
+
   border-bottom-color: #adadad;
 }
 
-#datausage-tab .toggle-data li > gaia-checkbox {
-  width: 100%;
-}
+#datausage-tab .toggle-data li label {
+  /* Removing BB absolute positioning styles */
+  position: unset;
 
-#datausage-tab .toggle-data li > gaia-checkbox > label {
-  width: 100%;
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+
+  width: 100%;
+  -moz-padding-end: 2.2rem;
+
+  pointer-events: auto;
 }
 
-#datausage-tab .toggle-data li > gaia-checkbox > label> span {
+#datausage-tab .toggle-data .data-origin-label,
+#datausage-tab .toggle-data .overview {
   font-size: 1.9rem;
   font-weight: normal;
   line-height: 3.8rem;
 }
 
-#datausage-tab .toggle-data li > gaia-checkbox > label> span:first-child {
+#datausage-tab .toggle-data .data-origin-label {
```

Comment:
Recently we were advised to get rid of ```:xxx-child``` selectors in SMS as it may affect reflow-performance, so I used chance to remove it here as well.